jekyll/_includes/social-buttons.html

```diff
@@ -12,8 +12,11 @@
     <a href="https://github.com/circleci" target="_blank" class="social-btn-github" title="GitHub">
       <i class="fa fa-github" aria-hidden="true"></i><span class="sr-only">GitHub</span>
     </a>
+    <a href="https://twitch.tv/circleci" target="_blank" class="social-btn-linkedin" title="LinkedIn">
```

Comment:
Not sure why these links have these classes, as they don't seem to be being used, but in any case I would recommend changing this class to `social-btn-twitch` and `title="Twitch"` so that it matches the content.

Reply:
And I'm not sure why I spaced on that lol

Reply:
@janedotbiz I have fixed that derp.